### ISSUE TITLE
Eventemmiter memory leak

### ISSUE THREAD

**Issue by gAmUssA** [mheap/pocket-tagger-cli#28]
Hello @mheap 
I'm having trouble processing tags for my pocket.

```
 pocket-tagger Waiting for all pages to be fetched + analysed +0ms
(node:70549) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 pipe listeners added to [Request]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)
(node:70549) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 pipe listeners added to [Request]. Use emitter.setMaxListeners() to increase limit
^C
```

after this message `pocket-tagger-cli` node process seems to hang. 

```
npm version
{
  npm: '9.4.0',
  node: '19.5.0',
  v8: '10.8.168.25-node.11',
  uv: '1.44.2',
  zlib: '1.2.13',
  brotli: '1.0.9',
  ares: '1.18.1',
  modules: '111',
  nghttp2: '1.51.0',
  napi: '8',
  llhttp: '8.1.0',
  uvwasi: '0.0.14',
  acorn: '8.8.1',
  simdutf: '3.1.0',
  undici: '5.14.0',
  openssl: '3.0.7+quic',
  cldr: '42.0',
  icu: '72.1',
  tz: '2022g',
  unicode: '15.0',
  ngtcp2: '0.8.1',
  nghttp3: '0.7.0'
}
```

❱ pocket-tagger --version
0.4.0

Not sure why pocket-tagger is choking on my account.

**Comment by mheap** [comment]
Can you run with debug enabled?

```bash
DEBUG="pocket*" pocket-tagger
```

**Comment by gAmUssA** [comment]
The output is too big - it spits out every URL it found in my account.

**Comment by mheap** [comment]
Does it end with something like this?

```
  pocket-tagger Waiting for all pages to be fetched + analysed +0ms
  pocket-tagger Persisting tags +6s
  pocket-tagger Error fetching URL 'https://example.com' +0ms
  pocket-tagger Error fetching URL 'https://other.example.com' +0ms
  pocket-tagger Sending tag updates +0ms
```

How many URLs are in your account?

**Comment by gAmUssA** [comment]
reeder shows 8k links in the account. no errors fetching url

**Comment by gAmUssA** [comment]
I see a bunch of warnings like 

```
WARNING: No stopwords file found for 'ja' - defaulting to English!
WARNING: No stopwords file found for 'ja' - defaulting to English!
```

**Comment by mheap** [comment]
Resolved via #29
